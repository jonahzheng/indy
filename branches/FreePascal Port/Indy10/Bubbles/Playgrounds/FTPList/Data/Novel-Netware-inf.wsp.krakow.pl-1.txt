-[R----F-]  1 raj           857 Feb 23  2000 !info!
d[R----F-]  1 raj           512 Nov 30  2001 incoming
d[R----F-]  1 raj           512 Mar 29  2000 novell
d[R----F-]  1 raj           512 May 16  2000 windows
d[R----F-]  1 raj           512 Mar 29  2000 dos
d[R----F-]  1 wojteksu      512 Mar 06 11:20 grudziu
