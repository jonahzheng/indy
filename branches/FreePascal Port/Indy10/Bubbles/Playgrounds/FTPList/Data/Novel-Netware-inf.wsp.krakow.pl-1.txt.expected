!info!
raj
857
File
0
0
2-23-2000

incoming
raj
512
Directory
0
0
11-30-2001

novell
raj
512
Directory
0
0
3-29-2000

windows
raj
512
Directory
0
0
5-16-2000

dos
raj
512
Directory
0
0
3-29-2000

grudziu
wojteksu
512
Directory
0
0
3-6-2002 11:20:00

